content/blog/adoptopenjdk-to-join-the-eclipse-foundation/index.md

```diff
@@ -0,0 +1,68 @@
+---
+title: AdoptOpenJDK to Join the Eclipse Foundation
+date: "2020-06-19T09:00:00+00:00"
+author: tsc
+description: AdoptOpenJDK will Join the Eclipse Foundation and rebrand as Adoptium.
+featuredImage: "./adopt_to_eclipse.png"
+tags:
+  - AdoptOpenJDK
+  - Eclipse
+  - Foundation
+---
+
+![Adopt to Eclipse Logo](./adopt_to_eclipse.png)
+
+Since early 2017 Java™ developers, vendors, Fortune 500™ companies, end-users, hobbyists, and a multitude of other folks from the Java ecosystem have all come together at [AdoptOpenJDK](https://adoptopenjdk.net/) to solve an industry challenge of
+
+_Providing the Java community with rock-solid runtimes and associated tools that can be used **free of charge, without usage restrictions on a wide range of platforms.**_
+
+AdoptOpenJDK has proven to be a runaway success, with [over 165M downloads](https://dash-v2.adoptopenjdk.net/) of our binary distributions now powering Java workloads across the globe. However, technology is not built in isolation, and we consider the greatest success of AdoptOpenJDK to be the coming together of hundreds of people in an active, diverse, and enthusiastic community.
+
+Currently, the AdoptOpenJDK project is legally backed by the non-profit [London Java Community CIC](https://londonjavacommunity.co.uk/) (aka the London Java User Group or LJC), which has served us well up to now. The LJC has kindly handled our accounts, signed contracts, and managed our legal obligations. However, now the time has come for AdoptOpenJDK to grow into its next stage of evolution. This next stage is a move to a large established Software Foundation.
```

Comment:
Is "Software Foundation" a name? I think it should be lower-cased, so: ... established **s**oftware **f**oundation.